tests/js/subject.pl

```diff
@@ -6,37 +6,37 @@
 
 property(c, "state").
 property_getter(c, Base, "state", Value) :- triple(Base, "todo://state", Value).
-property_setter(c, "state", '[{action: "setSingleTarget", source: "this", predicate: "todo://state", target: "value"}]').
+property_setter(c, "state", '[{action: "setSingleTarget", source: "this", predicate: "todo://state", target: "value", local: false}]').
```

Comment:
These `local: false` mentioned shouldn't be needed. **Social** DNA is about shared data anyways. Adding local properties to our SDNA tooling is for dev convenience and makes sense, but let's have shared stay the default in the normative SDNA definition (i.e.: here).